From 590c4a20e09e8ce0af7ae3b85229a8436363a725 Mon Sep 17 00:00:00 2001
From: "wenbiao.zhang" <wenbiao.zhang@amlogic.com>
Date: Thu, 21 Aug 2014 19:23:45 +0800
Subject: [PATCH 4741/5965] PD #96325: disable
 CONFIG_BCMDHD_POWER_OFF_IN_SUSPEND in meson8_tee_defconfig

---
 arch/arm/configs/meson8_tee_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/meson8_tee_defconfig b/arch/arm/configs/meson8_tee_defconfig
index 95b39a383bb9..dd784489aa0f 100755
--- a/arch/arm/configs/meson8_tee_defconfig
+++ b/arch/arm/configs/meson8_tee_defconfig
@@ -218,7 +218,6 @@ CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
 CONFIG_AM_WIFI_USB=y
 CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
-CONFIG_BCMDHD_POWER_OFF_IN_SUSPEND=y
 CONFIG_RTL8188EU=m
 CONFIG_RTL8188EU_MP=m
 CONFIG_BT_DEVICE=y
-- 
2.19.0

